src/main/gov/nasa/jpf/jdart/constraints/ConstraintsTree.java

```diff
@@ -107,6 +109,50 @@ private String toString(final String prefix, boolean values, boolean postconditi
                     
       return ret;
     }
+
+    public void toJson() {
+      toJson(false, true);
+    }
+
+    private void toJson(boolean values, boolean postconditions) {
+      String tree = toJson("", "", values, postconditions);
+      
+      try(PrintWriter out = new PrintWriter("tree.json")){
```

Comment:
The output path should be made optional---it should be possible to specify it in the jpf file